### PR TITLE
Supercollective stats now include sub collectives

### PR DESCRIPTION
**Pull request by xdamman**
Though it's definitely slower than before. We should cache this better.

### CODE REVIEW

File: server/models/Group.js [hunk 1]
```diff
@@ -220,6 +221,22 @@ export default function(Sequelize, DataTypes) {
     },
 
     instanceMethods: {
+      getSuperCollectiveGroupsIds() {
+        if (!this.isSupercollective) return Promise.resolve([this.id]);
+        if (this.superCollectiveGroupsIds) return Promise.resolve(this.superCollectiveGroupsIds);
```
Comment (asood123) [inline server/models/Group.js]: What's this supposed to do?

Comment (asood123) [inline server/models/Group.js]: is this like a poor man's cache? :)

**Comment by asood123** [comment]
Did you test how much slower this is for opensource collective?

**Comment by xdamman** [comment]
Keep in memory the list of ids so that we don't have to compute them again
(since we do multiple calls to get total backers, total donations, etc.)

**Pull request merged**